docs/metrics/README.md

```diff
@@ -6,8 +6,3 @@ This directory contains auto-generated documentation for all such metrics.
 Products that send telemetry via Glean *must request* a data-review following
 [the Firefox Data Collection process](https://wiki.mozilla.org/Firefox/Data_Collection)
 before integrating any of the components listed below.
-
```

Comment:
Please also remove line 4 which now falsely claims that "this directory contains auto-generated documentation". I actually wonder if we should just entirely delete this directory, but that's probably bigger than this PR.

Reply:
Yeah doesn't make sense to be a directory with just the README. Replacing the directory with a `metrics.md` was simple enough, so opted to do that! 😄